src/stylesheets/core/_utilities.scss

```diff
@@ -16,6 +16,13 @@
   left: -999em;
 }
 
+// Aria hidden helper
+@mixin aria-hidden() {
```

Comment:
I think when I said a global utility, I mean a class, such as `.hidden`. Sorry for the confusion. Although, what might be the benefits of using a mixin like this? It might be a better solution.